rsocket/framing/FrameTransportImpl.cpp

```diff
@@ -20,7 +20,7 @@ FrameTransportImpl::FrameTransportImpl(
 }
 
 FrameTransportImpl::~FrameTransportImpl() {
-  VLOG(1) << "~FrameTransport";
+  VLOG(1) << "~FrameTransport (" << this << ")";
```

Comment:
nit: Is it safe to call `operator<<` of `this` at the destructor? general question.

Reply:
the usage here should be fine.

Reply:
it prints out only address so its always safe.